pom.xml

```diff
@@ -2362,13 +2362,17 @@ under the License.
 								<!--<include>@org.apache.flink.annotation.PublicEvolving</include>-->
 							</includes>
 							<excludes>
+								<exclude>@java.lang.Deprecated</exclude>
+								<exclude>*.scala</exclude>
 								<exclude>@org.apache.flink.annotation.Experimental</exclude>
 								<exclude>@org.apache.flink.annotation.PublicEvolving</exclude>
 								<exclude>@org.apache.flink.annotation.Internal</exclude>
 								<!-- MARKER: start exclusions; these will be wiped by tools/releasing/update_japicmp_configuration.sh -->
 								<!-- Mark these 2 methods to @Internal. Tracked under FLINK-34130, should be removed in 2.0 -->
 								<exclude>org.apache.flink.configuration.Configuration#getBytes(java.lang.String,byte[])</exclude>
 								<exclude>org.apache.flink.configuration.Configuration#setBytes(java.lang.String,byte[])</exclude>
+								<!-- FLINK-35886: WatermarksWithIdleness constructor was marked as deprecated -->
+								<exclude>org.apache.flink.api.common.eventtime.WatermarksWithIdleness</exclude>
```

Comment:
I struggled to get my head around why this exclusion was necessary. It's a change that was introduced in 2.0 (see FLINK-35886) which deprecated the constructor.

My reasoning now is that the reference data (that was generated as part of the 1.20 release branch creation) didn't include that deprecation, yet. Excluding deprecated APIs now makes them be ignored in the check, i.e. the API disappears from the compare-to set (master HEAD) but is still present in the base set (1.20.0). :thinking: